.github/workflows/release.yml

```diff
@@ -54,56 +57,113 @@ jobs:
         name: enarx-${{ matrix.platform.target }}-oci
         path: enarx-${{ matrix.platform.target }}-oci
 
-  test-bin:
+  windows-build:
+    name: enarx Windows build
+    runs-on: windows-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup Rust toolchain
+        run: rustup show
+      - uses: actions-rs/cargo@v1
+        with:
+          command: build
+          args: --release
+      - uses: actions/upload-artifact@v3
+        with:
+          name: enarx-x86_64-windows
+          path: target/release/enarx.exe
+
+  universal-binary:
     needs: build
+    runs-on: macos-latest
+    steps:
+    - uses: actions/download-artifact@v3
+      with:
+        name: enarx-aarch64-apple-darwin
+    - uses: actions/download-artifact@v3
+      with:
+        name: enarx-x86_64-apple-darwin
+    - run: lipo -create ./enarx-aarch64-apple-darwin ./enarx-x86_64-apple-darwin -output ./enarx-universal-darwin
+    - uses: actions/upload-artifact@v3
+      with:
+        name: enarx-universal-darwin
+        path: enarx-universal-darwin
+
+  test-bin:
+    needs: universal-binary
     strategy:
       matrix:
         platform:
-        - os: macos-11
+        - host: macos-latest
           target: x86_64-apple-darwin
 
-        - os: ubuntu-20.04
+        - host: ubuntu-20.04
           target: x86_64-unknown-linux-musl
 
+        - host: aarch64-apple-darwin
+          target: aarch64-apple-darwin
+
+        - host: macos-latest
+          target: universal-darwin
+
+        - host: aarch64-apple-darwin
+          target: universal-darwin
+
         # TODO: Enable aarch64
```

Comment:
Is this TODO no longer necessary?

Reply:
I think it is still required for linux aarch64.